integrationtests/src/test/java/org/axonframework/integrationtests/testsuite/administration/commands/CreateEmployee.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2010-2025. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.integrationtests.testsuite.administration.commands;
+
+import org.axonframework.integrationtests.testsuite.administration.common.PersonIdentifier;
+import org.axonframework.modelling.annotation.TargetEntityId;
+
+public record CreateEmployee(
+        @TargetEntityId
+        PersonIdentifier identifier,
+        String lastNames,
+        String firstNames,
```

Comment:
```suggestion
        String lastName,
        String firstName,
```

Did you want that?

Reply:
In all Spanish-speaking countries, it's common to have two first names and two last names.
For example, Hilka Sofia Gonzalez Garcia. I actually have found unusable by me because I only have one :P